### DISCUSSION TITLE
CSS working in localhost, but not production

### DISCUSSION THREAD

**Discussion by audn** [tailwindlabs/tailwindcss#2757] [Help] [answered]
Hey, this repo (https://github.com/audn/frontend-boilerplate) works fine in my localhost, but not after hooking it up to Vercel.
Any idea why?

To reproduce, just `yarn` and `yarn dev` and it'll work like a charm, but on vercel it's just "nah fam"

**Comment by audn** [comment]
Demo: https://boilerplate-demo.vercel.app/

**Comment by simonswiss** [comment, marked as answer]
Hey - haven't had a look at your repo in details yet, but I'd try remove the purge config from the `postcss.config.js` file, since this is handled via the Tailwind config now.

If that doesn't work, I'll have a look in the morning (it's 8pm here in Australia) 👍

**Reply by audn** [reply]
Hey, this didn't seem to work, unless I did it wrong.
No worries, take a look when you have time man :D

**Reply by audn** [reply]
Solved it! Just removed purge from tailwind.config.js and added it back into postcss.config.js :D

**Reply by simonswiss** [reply]
Ok so I couldn't resist taking a look at your repo.

After removing the purge stuff from your PostCSS config, the `yarn build` command doesn't strip out the CSS from Tailwind:

```diff
module.exports = {
  plugins: {
    tailwindcss: {},
    'postcss-preset-env': {},
    'postcss-nested': {},
    ...(process.env.NODE_ENV === `production`
      ? {
-         '@fullhuman/postcss-purgecss': {
-           content: [`./src/**/*.js`],
-           defaultExtractor: (content) =>
-             content.match(/[A-Za-z0-9-_:/]+/g) || []
-         },
          autoprefixer: {}
        }
      : {})
  }
}
```

That should solve your problem 👍

**Reply by audn** [reply]
Pardon the late reply! I was looking through my old discussions and I saw this.

It was fixed by changing to `purge: false` in tailwind.config.js
